jans-cedarling/cedarling_opa/convert.py

```
@@ -0,0 +1,228 @@
+    aggregated = {}
+    if not dir_path.exists():
+        return aggregated
+
+    for entry in dir_path.iterdir():
+        if entry.is_file() and entry.name.endswith(".json"):
+            with open(entry, encoding="utf-8") as f:
+                try:
+                    content = json.load(f)
+                    aggregated[entry.stem] = content
+                except json.JSONDecodeError:
+                    raise ValueError(f"Invalid JSON file {entry.name}")
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)